Documentation/op-guide/performance.md

```diff
@@ -10,4 +10,26 @@ There are other sub-systems which impact the overall performance of etcd. Each s
 
 ## Benchmarks
 
-TODO
\ No newline at end of file
+To benchmark an indivisual package, etcd has Go benchmark tests. For example, mvcc package has `BenchmarkWatchableStoreWatchSyncPut` to measure how fast multiple synced watchers can receive events. To learn more about `go test` command, please visit [testing](https://golang.org/pkg/testing/). To benchmark etcd performance, simply use [benchmark](https://github.com/coreos/etcd/tree/master/tools/benchmark) CLI tool. Following results are run with:
```

Comment:
remove the package benchmark. the target audience here is ops. just take about the etcd cluster performance.